contents/blog/sales-no-experience.md

```diff
@@ -0,0 +1,142 @@
+---
+date: 2024-09-30
+title: How to do sales with no experience
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+---
+
+"I just need to hire a sales person so we get some revenue". There it is people, the start of the end.
+
+Here's the rest of the movie:
+
+* salesperson starts
+* they get a few meetings, they're pretty random feeling
+* you start building stuff that people in the meetings have talked about
+* nothing happens for ages, or at all
+* you now don't know if the salesperson is at fault, or if you don't have product market fit
+
+## Who I'm writing this for
+
+Are you the founder of a startup that has never sold before, and you're trying to figure out product market fit? This is written for you.
+
+## One job, people
+
+You have one job. Get to product market fit.
+
+It's a _terrible_ idea to out
+
+It _could_ of course work, but it just adds another variable to the most important and most multivariate problem you will _ever_ solve as a startup.
+
+## A little pep talk
+
+_You_ are the best salesperson for your product. There will _never_ be anyone else as well qualified to sell.
+
+* You get the _why_ behind what you're building
+* You are the parent of your startup (TODO INSERT LINK TO AIRBNB FOUNDER MODE BIT ON THIS). This means you are able to tweak things - like shipping an extra feature on the spot, your pricing, or how you pitch the product.
+
+You need to prioritize figuring out making money as one of the very top things:
+
+* do users like your product
+* do users pay for it
+
+Everything else is a distraction early on. So what are you doing if you don't care about this? Likely being avoidant from all the rejection and fear of failure! You may not want to hear it but a magical salesperson will be _far_ less likely to figure this out than you.
+
+## Getting meetings
```

Comment:
I feel like there should be a tiny transition from:

> * do users like your product
> * do users pay for it

To  "getting meetings", something like "at the beginning, the only way to get people to pay for your product is to ask them, you need to talk to them, and that requires meetings"